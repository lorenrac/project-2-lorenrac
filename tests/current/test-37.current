ERROR (line 4): Redeclaration of variable 'test' (originally defined on line 2)
