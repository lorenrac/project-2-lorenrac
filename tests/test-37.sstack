// ERROR: Redeclaring a variable after entering and leaving another scope.
VAR test="Test!"
IF (test > "A") {
  VAR test = "Okay redeclaration in a new scope."  
}
VAR test="Bad redeclaration"
