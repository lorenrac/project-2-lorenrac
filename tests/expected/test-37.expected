ERROR (line 6): Redeclaration of variable 'test' (originally defined on line 2)
